app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -777,14 +778,24 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
             if (browserFragment != null) {
                 browserFragment.setContentBlockingEnabled(turboEnabled);
             }
-            menu.findViewById(R.id.menu_turbomode).setSelected(turboEnabled);
+            if (menu != null) {
```

Comment:
Consider making this complex operation a function that takes int id as parameter.